projects/wave-core-new/src/lib/operators/dialogs/temporal-raster-aggregation/temporal-raster-aggregation.component.html

```diff
@@ -25,6 +25,14 @@
             <input matInput type="number" placeholder="Window Size" formControlName="windowSize" />
         </mat-form-field>
 
+        <div>
+            <mat-checkbox formControlName="windowReferenceChecked">Specify a custom point in time for the aggregation window</mat-checkbox>
```

Comment:
I think more information is needed here. So maybe add what the "anchor" point does.